CONTRIBUTING.md

```diff
@@ -91,7 +91,7 @@ ctest -V
 ```
 
 
-### Test with mocked functions
+### Test With Mocked Functions
```

Comment:
With this style, `with` is capitalized but not `and`? interesting.

Reply:
I agree with Ahson in favor of Test with mocked functions.

Reply:
works for me. So all "with" lowercase?

Reply:
https://academicguides.waldenu.edu/writingcenter/apa/other/capitalization#s-lg-box-2805312
https://academicguides.waldenu.edu/writingcenter/apa/other/headinglevels

> Major Words in All Headings

> In title case, the first letters of words with four or more letters are capitalized, while all other letters are left lowercase.

Prepositions shouldn't be capitalized. See https://capitalizemytitle.com/

So, I think `with` should remain lower case.

Reply:
Here/elsewhere.
```suggestion
### Test with Mocked Functions
```

Reply:
Yea I used that exact website. If you paste in "Test With Mocked Functions" every one of them capitalizes except for Chicago Style

Reply:
It's also "four or more" so it's capitalized. Interesting to know why. We can go either way I don't really midn

Reply:
Wikipedia style says 5 or more (on that website) :p
> Prepositions that contain five letters or more.

https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Titles#Capital_letters

Reply:
Haha this is why I didn't want to be an English major. Okay so I will change it to lowercase for "with"